File: GeeksForGeeks-Problems/C/Flow Control/The FizzBuzz Program.c
//User function Template for C

void fizzBuzz(int number){
    // Write your code here.
    if(number%3==0 & number%5==0){
        printf("FizzBuzz\n");
    }
    else if(number%3==0){
        printf("Fizz\n");
    }
    else if(number%5==0){
        printf("Buzz\n");
    }
    else 
    printf("%d\n",number);
}

File: GeeksForGeeks-Problems/JAVA/Flow Control/The FizzBuzz Program.java
import java.util.*;

public class Solution {
    public static void main(String args[]) {
        // Your Code Here
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        if (a % 3 == 0) {
            System.out.print("Fizz");
            if (a % 5 == 0) {
                System.out.print("Buzz");
            }
            System.out.println();
        } else if (a % 5 == 0) {
            System.out.println("Buzz");
        } else {
            System.out.println(a);
        }
    }
}